recipes/perl-uri/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "perl-uri" %}
+{% set version = "5.17" %}
+{% set sha256 = "5f7e42b769cb27499113cfae4b786c37d49e7c7d32dbb469602cd808308568f8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://cpan.metacpan.org/authors/id/O/OA/OALDERS/URI-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [win]
+  run_exports:
+    weak:
+      - {{ name }} ={{ version }}
+
+requirements:
+  build:
+    - make
+  host:
+    - perl
+    - perl-test-needs
+    - perl-test-warnings
+    - perl-test-fatal
+  run:
+    - perl
+    - perl-business-isbn
```

Comment:
Please also remove the core module dependencies here. Perl and Busines::ISBN should be the only requirements you need. You can also check the current core module list here: https://perldoc.perl.org/modules

Reply:
done - sorry for the misunderstanding